### DISCUSSION TITLE
Hot to cite Mido?

### DISCUSSION THREAD

**Discussion by pamonroy** [mido/mido#597] [General]
I would like to cite Mido in a scientific paper.  How should I do it?

**Comment by rdoursenaud** [comment]
Hi, thanks for reaching out and sorry for the late reply.

I don’t think we have any specific requirements.

@olemb is the main author. I (@rdoursenaud) maintain the project with the assistance of @bast.
Copyright and licensing information is provided in [reuse](https://reuse.software/) compliant headers for each file.
You may link to this GitHub repository and/or our [documentation](https://mido.readthedocs.io/).

If these informations are not enough, could you please elaborate on what would be required from us?

Thanks.
